src/modules/keyboardmanager/ui/KeyDropDownControl.cpp

```diff
@@ -3,12 +3,37 @@
 #include "keyboardmanager/common/Helpers.h"
 #include <keyboardmanager/common/KeyboardManagerState.h>
 #include "BufferValidationHelpers.h"
+#include <common\shared_constants.h>
 
 // Initialized to null
 KeyboardManagerState* KeyDropDownControl::keyboardManagerState = nullptr;
 
+// Get keys code list depending if Disable is in dropdown
+std::vector<DWORD> KeyDropDownControl::GetKeyCodeList(bool isShortcut, bool renderDisable)
+{
+    auto list = keyboardManagerState->keyboardMap.GetKeyCodeList(isShortcut);
+    if (renderDisable)
+    {
+        list.insert(list.begin(), CommonSharedConstants::VK_DISABLED);
+    }
+
+    return list;
+}
+
+// Get keys name list depending if Disable is in dropdown
+std::vector<std::wstring> KeyDropDownControl::GetKeyNameList(bool isShortcut, bool renderDisable)
+{
+    auto list = keyboardManagerState->keyboardMap.GetKeyNameList(isShortcut);
+    if (renderDisable)
+    {
+        list.insert(list.begin(), L"Disable");
```

Comment:
I think it should be one name. Disable fits better here because it is a verb. I will not move it to resx file to be consistent with keyboardLayoutMap initialization